### DISCUSSION TITLE
What kind of sports and activity do you like?

### DISCUSSION THREAD

**Discussion by yuki3406** [NAIST-SE/AttractingContributors#1095] [2024 Attraction Contributions]
I like soccer.

https://github.com/yuki3406/AttractingContributors

**Comment by gill876** [comment]
I like skiing!
Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking

**Comment by KazumaYamasaki** [comment]
I added to Baseball to your README.md
I would appreciate it if you could also make a pull request to [my repository](https://github.com/KazumaYamasaki/AttractingContributors-Keyboard) !
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard

**Comment by Ashmari** [comment]
Hi, I added mine .
I would be happy if you could also send a pull request to my repository!

https://github.com/Ashmari/AttractingContributors_BookVoyage.git

**Comment by mustafizur-r** [comment]
Hello [@yuki3406](https://github.com/yuki3406) , I recently made a contribution to your project , I have added “Cricket" could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.
